contracts/0.7.x/libraries/Binary/BinaryArbitrable.sol

```diff
@@ -219,7 +195,7 @@ library BinaryArbitrable {
         ArbitrableStorage storage self, 
         uint256 _disputeID, 
         uint256 _ruling
-    ) internal returns(Party finalRuling) {
+    ) internal returns(uint256 finalRuling) {
```

Comment:
The docs of this method:
> Validates and registers the ruling for a dispute. Can only be called by the arbitrator.

I think the latest part ("Can only be called by the arbitrator.") shouldn't be here, right?

Reply:
Hmmm, maybe "Should be called by the arbitrator when rule() is called" or sth like that?

Reply:
I mean, if it is not called by the arbitrator it reverts

Reply:
"Should be called from rule() when the arbitrator gives a ruling"?